docs/chapters/05_going-deeper.md

```diff
@@ -0,0 +1,367 @@
+
+Currently, the Booster framework provides a fully working provider package:
+
+-  **framework-provider-aws-\***
+
+Other providers packages are currently under experimental support. Some of the features might be missing:
+
+- **framework-provider-kubernetes-\***
+- **framework-provider-azure-\***
+
+## Configuration and environments
+
+Booster uses sensible defaults, convention over configuration, and code inference to reduce dramatically the amount of configuration needed. However, there are some aspects that can't be inferred (like the application name) or the provider library used for each [environment](#environments).
+
+### Booster configuration
+
+You configure your application by calling the `Booster.configure()` method. There are no restrictions about where you should do this call, but the convention is to do it in your configuration files located in the `src/config` folder. This folder will get automatically generated for you after running the `boost new:project <project-name>` CLI command.
+
+This is an example of a possible configuration:
+
+```typescript
+import { Booster } from '@boostercloud/framework-core'
+import { BoosterConfig } from '@boostercloud/framework-types'
+import * as AWS from '@boostercloud/framework-provider-aws'
+
+Booster.configure('pre-production', (config: BoosterConfig): void => {
+  config.appName = 'my-app-name'
+  config.provider = AWS.Provider
+})
+```
+
+The following is the list of the fields you can configure:
+
+- **appName:** This is the name that identifies your application. It will be used for many things, such us prefixing the resources created by the provider. There are certain restrictions regarding the characters you can use: all of them must be lower-cased and can't contain spaces. Two apps with different names are completely independent.
+
+- **provider:** This field contains the provider library instance that Booster will use when deploying or running your application.
+
+> **Note:** So far, there is only one provider fully supported in Booster yet, @boostercloud/framework-provider-aws, and it is probably the one you have already set if you used the generator to create your project. Azure and Kubernetes have experimental support and the team is currently improving them.
+
+- **assets**: This is an array of _relative_ paths from the root of the project pointing to files and folders with static assets. They will be included among the deployed files to the cloud provider.
+  For example, imagine you are using the `dotenv` module so that all the environment variables you have in your `.env` files are loaded into memory in runtime. In order for this to work, you need to include your `.env` files as assets of your project, so that they are included when deploying. Assuming you only have a `.env` file in the root of your project, you should add the following to your configuration:
+  ```typescript
+  config.assets = ['.env']
+  ```
+
+# Providers configuration
```

Comment:
I believe here it should be `### Providers configuration`, don't you think? The doc section is being displayed wrongly in docsify due to this typo